chain/default_syncer.go

```diff
@@ -110,7 +110,7 @@ func (syncer *DefaultSyncer) getBlksMaybeFromNet(ctx context.Context, blkCids []
 // that interacts with the network. It does NOT add tipsets to the chainStore..
 func (syncer *DefaultSyncer) collectChain(ctx context.Context, tipsetCids types.SortedCidSet) ([]types.TipSet, error) {
 	var chain []types.TipSet
-	defer logSyncer.Info("chain synced")
+	defer logSyncer.Infof("chain fetch from network complete for %v", tipsetCids)
```

Comment:
This will always show the base tipset they had prior to chain fetching due to the value being overridden on 149 I believe, not where they are after collectionChain returns (input value).

```
chain = append([]types.TipSet{ts}, chain...)
tipsetCids, err = ts.Parents()
```

Reply:
doh you are right (`defer`), good catch